WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -90,24 +90,31 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 					trackedWallets.Remove(finishedCoinJoin.Wallet.WalletName);
```

Comment:
The return value is not checked. It may do some good to check if return value is `false` and if it is, then log an error ("this should never happen"?)

Reply:
Done.